exercise/templates/exercise/staff/_assessment_panel.html

```diff
@@ -19,7 +19,23 @@
 				{% endif %}
 			</div>
 			<div>
-				<button
+				{% if submission.late_penalty_applied %}
+				<form method="post" action="{{ submission|url:'submission-conversion' }}" style="display: inline-block">
+					{% csrf_token %}
+					<button
+						class="aplus-button--secondary aplus-button--sm aplus-button--left"
+						type="submit"
+					>
+						{% translate 'CONVERT_SUBMISSION' %}
+					</button><button
+						data-toggle="modal"
+						data-target="#resubmit-modal"
+						class="aplus-button--secondary aplus-button--sm aplus-button--right"
+						type="button"
+						title="{% translate 'HELP' %}"
+					>?</button>
+				{% endif %}
+				</form>				<button
```

Comment:
line break missing here, between `</form>` and `<button>`

Reply:
Thanks, will fix